### PR TITLE
removing the extra string if the student didnt answer any question

### PR DESCRIPTION
**Pull request by ahmedqarmout2**
I just removed the extra text

### CODE REVIEW

**Review by Mohamed-A-Mohamed — APPROVED** [review]
Nice work

**Review by esteeroramy — APPROVED** [review]

File: views/questionlist.pug [hunk 1]
```diff
@@ -1,20 +1,15 @@
...
+    li.collection-item.avatar
+      if q.type == 0
+        i.material-icons.circle font_download
+      else
+        i.material-icons.circle format_list_bulleted
+      span.title #{q.title}
+      p
+        | #{q.topic}
+        br
+        |          #{q.points}
```
Comment (esteeroramy) [inline views/questionlist.pug]: weird space here

Comment (ahmedqarmout2) [reply views/questionlist.pug]: fixed

**Review by Mohammed-JS-Khan — APPROVED** [review]
Good fix 👍

**Pull request merged**